forms-flow-web/src/components/FOI/customComponents/Attachments/AttachmentModal.js

```diff
@@ -315,21 +350,32 @@ export default function AttachmentModal({
                   totalUploadedRecordSize={totalUploadedRecordSize}
                   totalRecordUploadLimit={totalRecordUploadLimit}
                 /> 
-                :
-                <ModalForRename modalFor={modalFor} newFilename={newFilename} updateFilename={updateFilename} errorMessage={errorMessage} extension={extension} />
+                : ''
+              }
+              {
+                modalFor === 'rename' ? <ModalForRename modalFor={modalFor} newFilename={newFilename} updateFilename={updateFilename} errorMessage={errorMessage} extension={extension} />
+                : ''
               }
             </DialogContentText>
           </DialogContent>
           <DialogActions>
+            {
+            modalFor === 'reclassify' ?
+              <button className={`btn-bottom btn-save ${classes.btnenabled}`} onClick={saveNewCategory}>
+                Reclassify
+              </button> : ''
+            }
             {
               modalFor === 'rename'?
               <button className={`btn-bottom btn-save ${classes.btnenabled}`} onClick={saveNewFilename}>
                 Save
-              </button>
-              :
+              </button> : ''
+            }
+             {
+              modalFor !== 'rename' && modalFor !== 'reclassify' ?
               <button className={`btn-bottom btn-save ${ isSaveDisabled() ? classes.btndisabled : classes.btnenabled }`} disabled={isSaveDisabled()} onClick={handleSave}>
                 {uploadFor === "email" ? "Save Changes" : "Continue"}
-              </button>
+              </button> : ''
```

Comment:
I refactored the ternary operator here to have more modular access to what is displayed based off of the modalFor value - please check to make sure this doesn't break anything though!

Reply:
Change ternaries to conditional rendering